sentry-core/src/main/java/io/sentry/core/util/StringUtils.java

```diff
@@ -19,4 +20,18 @@ private StringUtils() {}
     }
     return null;
   }
+
+  /**
+   * Returns a Capitalized String eg session = Session
```

Comment:
`l`: Maybe we could be a little bit more specific here. 
`Changes the first char to upper case and all remaining chars to lower case. eg seSSioN = Session`

Reply:
will do it